python/llm/dev/benchmark/all-in-one/README.md

```diff
@@ -1,7 +1,7 @@
 # All in One Benchmark Test
 All in one benchmark test allows users to test all the benchmarks and record them in a result CSV. Users can provide models and related information in `config.yaml`.
 
-Before running, make sure to have [bigdl-llm](../../../README.md) installed.
+Before running, make sure to have [bigdl-llm](../../../README.md) and [bigdl-nano](../../../../nano/README.md) installed.
```

Comment:
do we really need `nano`?

Reply:
I think if we add `bigdl-llm-init` for BigDL-LLM, there should be no need for nano.

Reply:
We can add the nano env into llm in a later PR.